tests/files/abundance.tsv

```diff
@@ -0,0 +1 @@
+organism	taxon_id

```

Comment:
That's weird content for a file called `abundance.tsv`. One would expect a (small) transcript abundance table like the ones that `kallisto` produces.

Reply:
Then call it something like `FILE_EMPTY_TABLE` (for the variable in the tests) and `empty_table.tsv` (for the filename itself). Then we won't have this sort of confusion.